packages/product-feed-luigis-box/src/Model/FeedItem/LuigisBoxProductFeedItem.php

```diff
@@ -11,21 +11,18 @@
 
 class LuigisBoxProductFeedItem implements FeedItemInterface
 {
-    public const UNIQUE_IDENTIFIER_PREFIX = 'product';
-    protected const SMALL_IMAGE_SIZE = 100;
-    protected const MEDIUM_IMAGE_SIZE = 200;
-    protected const LARGE_IMAGE_SIZE = 600;
-    protected const AVAILABILITY_RANK_OUT_OF_STOCK = 15;
-    protected const AVAILABILITY_RANK_IN_STOCK = 1;
-    protected const AVAILABILITY_RANK_AVAILABLE_IN_LONG_TIME = 14;
+    public const string UNIQUE_IDENTIFIER_PREFIX = 'product';
+    protected const int SMALL_IMAGE_SIZE = 100;
+    protected const int MEDIUM_IMAGE_SIZE = 200;
+    protected const int LARGE_IMAGE_SIZE = 600;
+    protected const int ORDERABLE_PRODUCT_AVAILABILITY = 1;
```

Comment:
orderable is different from sellable?

Reply:
nope, I was just using the terminology from luigis box. However, changed to "sellable" in a1241f780c